--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -18,7 +18,7 @@
 find_package(ament_cmake REQUIRED)
 find_package(pluginlib REQUIRED)
 find_package(rcutils REQUIRED)
-find_package(yaml_cpp_vendor REQUIRED)
+find_package(yaml-cpp REQUIRED)
 
 add_library(
   rosbag2_storage
@@ -27,11 +27,12 @@
   src/rosbag2_storage/ros_helper.cpp
   src/rosbag2_storage/storage_factory.cpp)
 target_include_directories(rosbag2_storage PUBLIC include)
+target_link_libraries(rosbag2_storage yaml-cpp)
 ament_target_dependencies(
   rosbag2_storage
   pluginlib
   rcutils
-  yaml_cpp_vendor)
+  yaml-cpp)
 
 # Causes the visibility macros to use dllexport rather than dllimport,
 # which is appropriate when building the dll but not consuming it.
@@ -56,7 +57,7 @@
 
 ament_export_include_directories(include)
 ament_export_libraries(${PROJECT_NAME})
-ament_export_dependencies(pluginlib yaml_cpp_vendor)
+ament_export_dependencies(pluginlib yaml-cpp)
 
 if(BUILD_TESTING)
   find_package(ament_lint_auto REQUIRED)
